articles/openshift/openshift-faq.md

```diff
@@ -10,196 +10,220 @@ ms.date: 11/04/2019
 
-## Are master and infrastructure nodes abstracted away as they are with Azure Kubernetes Service (AKS)?
+Planned maintenance is not prescheduled with the customer. Notifications may be sent via email if communication to the customer is required.
 
-No. All resources, including the cluster master, run in your customer subscription. These types of resources are put in a read-only resource group.
+### What is the Azure Red Hat OpenShift (ARO) maintenance process?
 
-## Is Open Service Broker for Azure (OSBA) supported?
+There are two types of maintenance for ARO: upgrades and cloud provider-initiated maintenance.
+- Upgrades include software upgrades and CVEs.
+- Cloud provider-initiated maintenance includes network, storage, and regional outages. The maintenance is dependent on the cloud provider and relies on provider-supplied updates.
 
-Yes. You can use OSBA with Azure Red Hat OpenShift. See [Open Service Broker for Azure](https://github.com/Azure/open-service-broker-azure#openshift-project-template) for more information.
+### What about emergency vs. planned maintenance windows?
 
-## I am trying to peer into a virtual network in a different subscription but getting `Failed to get vnet CIDR` error.
+We do not distinguish between the two types of maintenance. Our teams are available 24/7/365 and do not use traditional scheduled “out-of-hours” maintenance windows.
 
-In the subscription that has the virtual network, make sure to register `Microsoft.ContainerService` provider with `az provider register -n Microsoft.ContainerService --wait` 
+### How will the host operating system and OpenShift software be updated?
 
-## What is the Azure Red Hat OpenShift (ARO) maintenance process?
+The host operating systems and OpenShift software are updated through our general upgrade process.
 
-There are three types of maintenance for ARO: upgrades, backup and restoration of etcd data, and cloud provider-initiated maintenance.
+### What’s the process to reboot the updated node?
 
-+ Upgrades include software upgrades and CVEs. CVE remediation occurs on startup by running `yum update` and provides for immediate mitigation.  In parallel a new image build will be created for future cluster creates.
+Nodes are rebooted as a part of an upgrade.
 
-+ Backup and management of etcd data is an automated process that may require cluster downtime depending on the action. If the etcd database is being restored from a backup there will be downtime. We back up etcd hourly and retain the last 6 hours of backups.
+## Cluster operations
 
-+ Cloud provider-initiated maintenance includes network, storage, and regional outages. The maintenance is dependent on the cloud provider and relies on provider-supplied updates.
+### Can I use Prometheus to monitor my applications?
 
-## What is the general upgrade process?
+Yes, you can deploy Prometheus in your namespace and monitor applications in your namespace.
```

Comment:
Can we link to prometheus or openshift docs on how to do this?

Reply:
I clarified the for 4.x Prometheus is built in and linked to the OpenShift docs with more info.

For 3.x I also added a link to the MSFT guide on this topic.